plugins/connection/network_cli.py

```diff
@@ -723,7 +723,7 @@ def receive_paramiko(
     ):
 
         recv = BytesIO()
-        cache_socket_timeout = self._ssh_shell.gettimeout()
+        cache_socket_timeout = self.get_option("persistent_command_timeout")
```

Comment:
We still need to set the timeout here, right?
```suggestion
        cache_socket_timeout = self.get_option("persistent_command_timeout")
        self._ssh_shell.settimeout(cache_socket_timeout)
```

With that, I think we can drop the other two `settimeout` calls below